src/stylesheets/_resource_view_more_link.scss

```diff
@@ -0,0 +1,37 @@
+.resource-view-more-link {
+    position: relative;
+    cursor: pointer;
+    height: 4.4rem;
+    width: 4.4rem;
+    text-align: center;
+    display: inline-block;
+    border-radius: 9999px;
+    background-color: transparent;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    z-index: 99;
+
+    &:hover,
+    &:focus {
+        background-color: $context-menu-color-hover;
+        color: $context-menu-color-trigger-hover;
+        transition: background-color 0.2s ease-out,
+            color 0.2s ease-out;
+    }
+
+    .icon {
```

Comment:
This could be done without nesting as well.

Reply:
Again, don't see what you mean, because we definitely don't want to apply that globally.

Reply:
You could add a more specific class to the icon like resource-tile-idontknow__icon, then it could be used without nesting.